logpyle/__init__.py

```diff
@@ -1262,6 +1262,7 @@ def __init__(self, mgr: LogManager, name: str = "t_log") -> None:
         self.log_manager = mgr
 
     def __call__(self) -> float:
+        add_hwm(self.log_manager)
         return self.log_manager.t_log
```

Comment:
Not the greatest place for this, but I couldn't find a better solution. In particular, `LogManager.close()` won't work since this is run from e.g. runalyzer-gather as well.

Reply:
Edit: Made it into a `LogQuantity` in [a634b74](https://github.com/illinois-ceesd/logpyle/pull/79/commits/a634b740ac8330d6b496e3aa8d351ac2f2ff5554)